### ISSUE TITLE
Feature Suggestion: C# String Encode

### ISSUE THREAD

**Issue by doggy8088** [madskristensen/Editorsk#1]
### Installed product versions
- Visual Studio: 2015 Enterprise
- This extension: 1.0.18
### Description

Add an C# String Encode 
### Steps to recreate
1. Open any text file in the editor
2. Make a selection on any range  of text
3. Mouse right click on the selection
4. Choose "Encode Selection", then click the "C# String Encode" (suggested feature)
### Current behavior

Don't have a "C# String Encode" feature.
### Expected behavior

Add "C# String Encode" feature.

**Comment by madskristensen** [comment]
What should it do? Escape quotation marks or...?

**Comment by doggy8088** [comment]
Yes.  Just escape the quotation marks or @.  But there are two types of string.
1. "xxxxx"
2. @"xxxxx"

Maybe you can provide two types of string encode feature.  Sometimes it just useful.

**Comment by wonderfuer** [comment]
wonderful
